src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -389,6 +389,22 @@ BOOST_AUTO_TEST_CASE(SimpleLinearLayerTest)
   BOOST_REQUIRE_EQUAL(arma::accu(delta), 0);
 }
 
+/**
+ * Linear module weight initialization test.
+ */
+BOOST_AUTO_TEST_CASE(LinearLayerWeightInitializationTest)
+{
+  size_t inSize = 10, outSize = 4;
+  Linear<>* linear = new Linear<>(inSize, outSize);
+  linear->Reset();
+  RandomInitialization().Initialize(linear->Weight(), outSize, inSize);
```

Comment:
Can you also check that `Parameters` is the same as `Weight + Bias` after you change the weights, just to make sure everything works as expected.

Reply:
Done.